cdap-proto/src/test/java/co/cask/cdap/proto/security/AuthorizableTest.java

```diff
@@ -31,15 +32,27 @@
   @Test
   public void testFromEntity() {
```

Comment:
Unless we have the conversion tests elsewhere, let's add conversion tests for the other entity types here. Also would be good to add the wildcards in the middle of the entity name, in the parent entity name, etc. to make sure all possible cases work.